zkstack_cli/crates/config/src/forge_interface/deploy_l2_contracts/input.rs

```diff
@@ -20,15 +20,18 @@ pub struct DeployL2ContractsInput {
 }
 
 impl DeployL2ContractsInput {
-    pub fn new(chain_config: &ChainConfig, era_chain_id: L2ChainId) -> anyhow::Result<Self> {
-        let contracts = chain_config.get_contracts_config()?;
+    pub fn new(
+        ecosystem_config: &EcosystemConfig,
+        chain_config: &ChainConfig,
+    ) -> anyhow::Result<Self> {
+        let contracts = ecosystem_config.get_contracts_config()?;
         let wallets = chain_config.get_wallets_config()?;
         Ok(Self {
-            era_chain_id,
+            era_chain_id: ecosystem_config.era_chain_id,
             chain_id: chain_config.chain_id,
             l1_shared_bridge: contracts.bridges.shared.l1_address,
             bridgehub: contracts.ecosystem_contracts.bridgehub_proxy_addr,
-            governance: wallets.governor.address,
+            governance: contracts.l1.governance_addr,
```

Comment:
i'd rather pass it as a parameter. We are doing some separation between chain and ecosystem and i don't want to make this dependency harder

Reply:
This is actually not a param, because this is the governance that is set for L2 bridges. The ecosystem governor should be the admin of those

Reply:
yeah, sure. 
i mean please do not pass the whole ecosystem